test_sets/Ideas_for_SDL_must_send_WARNINGS_success_true_to_mobile_app_if_HMI_respond_WARNINGS_at_least_to_one_component_of_RPC.txt

```diff
@@ -0,0 +1,29 @@
+
+
+*****Precondition: HMI does NOT respond to Interface.IsReady
+./test_scripts/API/AlertManeuver/ATF_TTS_IsReady_no_resp_AlertManeuver_TTS_Speak_UNSUP_RES.lua Precondition_HMI_does_not_respond_to_TTS_IsReady_mobile_app_sends_RPC_AlertManeuver_and_HMI_responds_to_Navi_AlertManeuver_with_SUCCESS_and_to_TTS_Speak_with_UNSUPPORTED_RESOURCE_success_true_SDL_should_sent_to_mobile_app_UNSUPPORTED_RESOURCE_success_true
+./test_scripts/API/SetGlobalProperties/ATF_TTS_IsReady_no_resp_SetGlobalProperties_TTS_SetGlobalProperties_gets_UNSUP_RES.lua Precondition_HMI_does_not_respond_to_TTS_IsReady_mobile_app_sends_RPC_SetGlobalProperties_and_HMI_responds_to_UI_SetGlobalProperties_with_SUCCESS_and_to_TTS_SetGlobalProperties_with_UNSUPPORTED_RESOURCE_success_true_SDL_should_sent_to_mobile_app_UNSUPPORTED_RESOURCE_success_true
+./test_scripts/API/AddCommand/ATF_VR_IsReady_no_resp_AddCommand_VR_AddCommand_UNSUP_RES.lua Precondition_HMI_does_not_respond_to_VR_IsReady_mobile_app_sends_RPC_AddCommand_and_HMI_responds_to_UI_AddCommand_with_SUCCESS_and_to_VR_AddCommand_with_UNSUPPORTED_RESOURCE_success_true_SDL_should_sent_to_mobile_app_UNSUPPORTED_RESOURCE_success_true
+./test_scripts/API/DeleteCommand/ATF_VR_IsReady_no_resp_DelCommand_VR_DelCommand_UNSUP_RES.lua Precondition_HMI_does_not_respond_to_VR_IsReady_mobile_app_sends_RPC_DeleteCommand_and_HMI_responds_to_UI_DeleteCommand_with_SUCCESS_and_to_VR_DeleteCommand_with_UNSUPPORTED_RESOURCE_success_true_SDL_should_sent_to_mobile_app_UNSUPPORTED_RESOURCE_success_true
+./test_scripts/API/PerformInteraction/ATF_VR_IsReady_no_resp_PerfInteract_VR_PerfInterac_UNSUP_RES.lua Precondition_HMI_does_not_respond_to_VR_IsReady_mobile_app_sends_RPC_PerformInteraction_and_HMI_responds_to_UI_PerformInteraction_with_SUCCESS_and_VR_PerformInteraction_with_UNSUPPORTED_RESOURCE_success_true_SDL_should_sent_to_mobile_app_UNSUPPORTED_RESOURCE_success_true
+./test_scripts/API/Alert/ATF_UI_IsReady_no_resp_UI_Alert_UNSUP_RES.lua Precondition_HMI_does_not_respond_to_UI_IsReady_mobile_app_sends_RPC_Alert_and_HMI_responds_to_TTS_Speak_with_SUCCESS_and_to_UI_Alert_with_UNSUPPORTED_RESOURCE_success_true_SDL_should_sent_to_mobile_app_UNSUPPORTED_RESOURCE_success_true
+./test_scripts/API/AddCommand/ATF_UI_IsReady_no_resp_UI_AddCommand_UNSUP_RES.lua Precondition_HMI_does_not_respond_to_UI_IsReady_mobile_app_sends_RPC_AddCommand_and_HMI_responds_to_VR_AddCommand_with_SUCCESS_and_to_UI_AddCommand_with_UNSUPPORTED_RESOURCE_success_true_SDL_should_sent_to_mobile_app_UNSUPPORTED_RESOURCE_success_true
+./test_scripts/API/DeleteCommand/ATF_UI_IsReady_no_resp_UI_DeleteCommand_UNSUP_RES.lua Precondition_HMI_does_not_respond_to_UI_IsReady_mobile_app_sends_RPC_DeleteCommand_and_HMI_responds_to_VR_DeleteCommand_with_SUCCESS_and_to_UI_DeleteCommand_with_UNSUPPORTED_RESOURCE_success_true_SDL_should_sent_to_mobile_app_UNSUPPORTED_RESOURCE_success_true

```

Comment:
@vvvakulenko, as CRQ is covering result code WARNINGS, VR_DeleteCommand should return WARNINGS.